Tools/autotest/vehicle_test_suite.py

```diff
@@ -3087,6 +3087,109 @@ def all_log_format_ids(self):
 
         return ids
 
+    def LoggerDocumentation_greylist(self):
+        '''returns a set of messages should should be documented but
+        are currently known as undocumented'''
+        return set([
+            "FENC",  # fence
+            "FTN3",  # gyrofft
+            "GEN",   # generator
+            "IE24",  # generator
+            "IEFC",  # generator
+            "IREG",  # INS something
+            "ISBD",  # InertialSensor batch logging
+            "ISBH",  # InertialSensor batch logging
+            "RELY",  # relay
+            "RTCM",  # GPS
+            "SBRE",  # septentrio
+
+            "SAF1",  # blimp-sim
+            "SAN1",  # blimp-sim
+            "SAN2",  # blimp-sim
+            "SBA1",  # blimp-sim
+            "SBLM",  # blimp-sim
+            "SCTL",  # glider-sim
+            "SFA1",  # blimp-sim
+            "SFAN",  # blimp-sim
+            "SFN",   # blimp-sim
+            "SFT",   # blimp-sim
+            "SFV1",  # blimp-sim
+            "SMGC",  # blimp-sim
+            "SRT1",  # blimp-sim
+            "SRT2",  # blimp-sim
+            "SRT3",  # blimp-sim
+            "SSAN",  # blimp-sim
+
+            "GLT",  # glider sim
+            "SL2",  # glider-sim
+            "SLD",  # glider-sim
+
+            "SMVZ",  # Sim-Volz
+
+            "SORC",  # Soaring
+            "VAR",   # soaring
+
+            "TCLR",  # tempcal
+            "TEMP",  # temperature sensor library
+
+            "ARHS",  # autorotation
```

Comment:
It's off topic but do we really have ARHS and AHRS?  I wonder how much we thought about this

Reply:
> It's off topic but do we really have ARHS and AHRS? I wonder how much we thought about this

:-)  It was like that when I got here :-)

That's be Auto Rotation Head Speed, at a guess.

Reply:
@peterbarker, yeah, I just had to say something.. marking this as "resolved"